docs/performance/quantization.md

```diff
@@ -31,45 +31,45 @@ Quantization in ONNX Runtime refers to 8 bit linear quantization of an ONNX mode
-  - Quantized models converted from tflite and other framework.
+  - Quantized models converted from TFLite and other frameworks.
 
-For the latter 2 cases, you don't need to quantize the model with quantization tool. OnnxRuntime can run them directly as quantized model.
+For the latter two cases, you don't need to quantize the model with the quantization tool. ONNX Runtime can run them directly as a quantized model.
 
-Picture below shows the equivalent representation with QDQ format and Operator oriented format for quantized Conv. This [E2E](https://github.com/microsoft/onnxruntime-inference-examples/tree/main/quantization/image_classification/cpu/run.py) example demonstrates QDQ and Operator Oriented format.
+The picture below shows the equivalent representation with the QOperator and QDQ formats for quantized Conv. [This end-to-end example](https://github.com/microsoft/onnxruntime-inference-examples/tree/main/quantization/image_classification/cpu/run.py) demonstrates the two formats.
 
 ![Changes to nodes from basic and extended optimizations](../../images/QDQ_Format.png)
 
 ## Quantizing an ONNX model
-There are 3 ways of quantizing a model: dynamic, static and quantize-aware training quantization.
+There are 3 ways of quantizing a model: dynamic, static and quantization-aware training quantization.
 
-* **Dynamic quantization**: This method calculates the quantization parameter (scale and zero point) for activations dynamically.
+* **Dynamic quantization**: This method calculates the quantization parameters (scale and zero point) for activations dynamically.
 
-* **Static quantization**: It leverages the calibration data to calculates the quantization parameter of activations.
+* **Static quantization**: leverages calibration data to calculate the quantization parameters of activations. Our quantization tool supports three calibration methods: MinMax, Entropy and Percentile. Please refer to [`calibrate.py`](https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/python/tools/quantization/calibrate.py) for details.
 
-* **Quantize-Aware training quantization**: The quantization parameter of activation are calculated while training, and the training process can control activation to a certain range.
+* **Quantization-Aware training quantization**: The quantization parameters of activations are calculated during training, and the training process can control activations to a certain range.
 
 ### Quantization API
 {: .no_toc}
 
-Quantization has 3 main APIs, which corresponds to the 3 quantization methods:
-* quantize_dynamic: dynamic quantization
-* quantize_static: static quantization
-* quantize_qat: quantize-aware training quantization
+Quantization has 3 main APIs, corresponding to the 3 quantization methods:
+* `quantize_dynamic`: dynamic quantization
+* `quantize_static`: static quantization
+* `quantize_qat`: quantization-aware training quantization
```

Comment:
quantize_qat was removed because Onnx Runtime can qat model directly now. Could you please also remove it?